### DISCUSSION TITLE
Get select label

### DISCUSSION THREAD

**Discussion by CFarhad** [mantinedev/mantine#3009] [Q&A]
Hi , I wanna get the select component label. I'm trying to get it with `useRef` but it doesn't work well.
what is the best way?

**Comment by rtivital** [comment]
Do you want to get label element DOM node?

**Comment by CFarhad** [comment]
@rtivital i dont know what you mean. but i want to get a select component label when it changes. a big problem is select will return a value and it is a number. how can i return the label ???

**Reply by rtivital** [reply]
`const selectedLabel = data.find(item => item.value === value)?.label`
